content/using/install.md

```diff
@@ -473,79 +294,9 @@ It’s a good idea to store your keys redundantly; for example, on both a USB st
-
-To start Screen, enter
-
-```
-screen
-```
-
-You'll be met with some copyright information. Hit enter or return to move past this info. Once you're inside Screen, you use `Ctrl-a` to tell it that you're about to enter a command. Screen is controlled through keyboard shortcuts. Here are some useful ones to be run inside a Screen:
-
-- To create a new Screen window, enter `Ctrl-a` and then `c`.
-- To exit a Screen window and go back to your regular Droplet terminal ("Detach" it), enter `Ctrl-a` and then `d`.
-- To see all Screen shortcuts, enter `Ctrl-a` and then `?`.
-
-There are a few useful Screen commands that can be run from your regular Droplet terminal.
-
-- To see all running Screen windows, enter `screen -ls`. This is useful if you're not sure if you're inside a Screen window; if you are, it will show a window as "Attached". (Note that you'll only ever need one screen for the purposes of hosting an Urbit remotely)
-- To return to the Screen window that you made ("Attach" it), enter `screen -r`.
-- To delete *all* Screen windows, enter `killall screen`.
-
-Now, let's put this information to use to boot your Urbit ship in a Screen window.
-
-1. Enter `cd urbit-v0.10.8-linux64` to change your directory to the folder containing the Urbit binary.
-2. Enter `screen -ls`. If you get anything but a message like `No Sockets found in /run/screen/S-root.`, run `killall screen` to wipe the slate clean.
-3. Enter `screen` and skip past the information. You're in an attached Screen window now; enter `screen -ls` to verify this.
-4. Open your keyfile on your local machine and copy the contents. On the Screen in your Droplet, enter `./urbit -w sampel -G rAnDoMkEy`, where `rAnDoMkEy` is the copied contents of the keyfile.
-
-Your ship should be booting, and displaying a series of boot messages starting with `urbit 0.10.8`. A directory called `sampel/` and begin building your ship. It may take a few minutes.
-
-When your ship is finished booting, you will see either the `~sampel:dojo>` or `~sampel:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-x` to switch into Dojo. At that point, you should permanently erase your keyfile from your machine.
-
-To shut down your ship, use `Ctrl-d`. Since you're running, you won't have much reason to do this often, since you should always have it running in your Droplet. More frequently you'll want to use `Ctrl-a` and then `d` to go back to your Droplet's command line while keeping your planet running on a detached Screen. Then, to return to your ship, run `screen -r`.
-
-If you need to shut down your ship, start it up again by running `./urbit sampel` from the directory where your Urbit binary is saved. Note that in this usage, `sampel/` is the path of a folder, which by default is located in the same folder as the Urbit binary. This folder is called your ship's **pier**.
+Because of their need for near-perfect uptime Stars should use cloud services. 
```

Comment:
```suggestion
Because of their need for near-perfect uptime Stars should be hosted on [cloud services](insert link here). 
```

Reply:
On this one I didn't include the link because the next line has the link to the guide. I did switch it to your language though.